ci-operator/step-registry/openshift-qe/day2-config/openshift-qe-day2-config-commands.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
At the moment this only supports deploying gateway_api, I think we should keep day-2 ops separate and avoid having multiple ones in the same script

Reply:
@rsevilla87 Yes, it only supports deploying gateway_api for now. I intended to make this step more portable to support more day-2 ops in the future, so that we can have only one step, just need pass env DAY2_CONFIG to decide which day-2 ops to install. Do you see any disadvantage? I can change it to a separate one.

Reply:
@rsevilla87 I added a new separate step gateway-api-day2-config